main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -177,6 +177,7 @@ class LanguageServer(port: Int) extends WebSocketServer(new InetSocketAddress("l
       case JString("lsp/hover") => Request.parseHover(json)
       case JString("lsp/goto") => Request.parseGoto(json)
       case JString("lsp/rename") => Request.parseRename(json)
+      case JString("lsp/symbols") => Request.parseSymbols(json)
```

Comment:
I would call it `lsp/documentSymbols` to avoid confusion. I would also call the method `parseDocumentSymbols`.